frontend/src/common/components/NavBarMain.tsx

```diff
@@ -83,90 +84,120 @@ const NavbarMain = () => {
+                  Donate
+                </Link>
               </Grid>
               {isSignedIn(user) ? (
                 <Grid item>
                   <div>
-                      <Button 
+                    <Button
                       sx={{
                         my: -0.75,
                         mx: -1,
-                      }}>
-                        <Typography
-                          onClick={handleOpenUserMenu}
-                          className="nav-link"
-                          style={{
-                            fontFamily: 'Lato, Arial, Helvetica, sans-serif',
-                            textTransform: 'none'
-                          }}
-                        >
-                          Account <ArrowDropDownIcon />
-                        </Typography>
-                      </Button>
+                      }}
+                    >
+                      <Typography
+                        onClick={handleOpenUserMenu}
+                        className="nav-link"
+                        style={{
+                          fontFamily: "Lato, Arial, Helvetica, sans-serif",
+                          textTransform: "none",
+                        }}
+                      >
+                        Account <ArrowDropDownIcon />
+                      </Typography>
+                    </Button>
                     <MenuList
                       anchorEl={anchorEl}
                       open={open}
                       onClose={handleCloseUserMenu}
                     >
                       <MenuItem>
-                        <Link href="/dashboard" id="basic-nav-dropdown">Dashboard</Link>
+                        <Link href="/dashboard" id="basic-nav-dropdown">
+                          Dashboard
+                        </Link>
                       </MenuItem>
                       <MenuItem>
-                        <Link href="/settings" id="basic-nav-dropdown">Settings</Link>
+                        <Link href="/settings" id="basic-nav-dropdown">
+                          Settings
+                        </Link>
                       </MenuItem>
                       <MenuItem divider>
-                        <Link href="/learn-mod" id="basic-nav-dropdown">Learn</Link>
+                        <Link href="/learn-mod" id="basic-nav-dropdown">
+                          Learn
+                        </Link>
                       </MenuItem>
                       <Link href={""} passHref id="basic-nav-dropdown">
                         <MenuItem
                           onClick={() => {
                             dispatch(signOutUser());
                           }}
                         >
-                          <Typography sx={{fontFamily:'Lato, Arial, Helvetica, sans-serif'}}>Log out</Typography>
+                          <Typography
+                            sx={{
+                              fontFamily: "Lato, Arial, Helvetica, sans-serif",
```

Comment:
This is supposed to be declared globally, in globals.css, which I think we've done. So you can remove this line